tests/server.test.js

```diff
@@ -0,0 +1,14 @@
+const test = require('tape');
+const server = require('../src/server');
+
+test('Check the index route', t => {
+  const options = {
+    method: 'GET',
+    url: '/'
+  };
+  server.inject(options, (response) => {
+    t.equal(response.statusCode, 200, 'You received a 200 status code, test passed');
+    server.stop();
```

Comment:
Okay I think you do not need this. In your project you have server.js file that does not start the server. Then you have start_server file. This is a really great pattern as it means you do not need to start / stop the server to run tests (Because Hapi is great and server.inject works without having to start the server!)